projects/sandbox/pyproject.toml

```diff
@@ -24,13 +24,13 @@ frame_type = "HOFT_C01"
 state_flag = "DCS-ANALYSIS_READY_C01:1"
 resnet = {layers = [3, 3, 3]}
 repository_directory = "${BASE_DIR}/model_repo/" 
-force_generation = false
```

Comment:
I think we want this as `false` in general, but I understand why you would want it to be `true` while you were testing this. If the tests take a while to run, we can ignore fixing this and I can fix it in one of my PRs today, or if this would be a quick turnaround we can go ahead and change it here. Up to you.

Reply:
Currently on a flight and can't connect to LIGO, so let's fix it in one of your PRs